scripts/reboot

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# Reboot immediately if we run the kdump capture kernel
+VMCORE_FILE=/proc/vmcore
+if [ -e $VMCORE_FILE -a -s $VMCORE_FILE ]; then
+        debug "We have a /proc/vmcore, then we just kdump'ed"
+        /sbin/reboot
```

Comment:
Is it possible that to call `/sbin/reboot` ahead of echoing `user request reboot` to /host/reboot-cause/reboot-cause.txt fails the reboot cause?